pkg/ebpf/processor_proctree.go

```diff
@@ -98,39 +98,6 @@ func (t *Tracee) procTreeForkProcessor(event *trace.Event) error {
 	)
 }
 
-// procTreeForkRemoveArgs removes arguments needed for the process tree only (when source is events).
-func (t *Tracee) procTreeForkRemoveArgs(event *trace.Event) error {
-	argsToRemove := []string{
-		"up_parent_tid",
-		"up_parent_ns_tid",
-		"up_parent_pid",
-		"up_parent_ns_pid",
-		"up_parent_start_time",
```

Comment:
If we make these fields permanent, we should rename them to something more clear.
I suggest calling it `parent_process_xxx` instead of `up_parent_xxx`

Reply:
Pls ref to https://github.com/aquasecurity/tracee/pull/4167#issuecomment-2204087350